docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -1974,6 +1975,58 @@ Office.context.mailbox.item.getInitializationContextAsync(
 ---
 ---
 
+#### getItemIdAsync([options], callback)
+
+Asynchronously gets the ID of a saved item. Compose mode only.
+
+When invoked, this method returns the item ID via the callback method.
+
+> [!NOTE]
+> If your add-in calls `getItemIdAsync` on an item in compose mode in order to get an `itemId` to use with EWS or the REST API, be aware that when Outlook is in cached mode, it may take some time before the item is actually synced to the server. Until the item is synced, using the `itemId` will return an error.
+
+##### Parameters
+
+|Name|Type|Attributes|Description|
+|---|---|---|---|
+|`options`|Object|&lt;optional&gt;|An object literal that contains one or more of the following properties.|
+|`options.asyncContext`|Object|&lt;optional&gt;|Developers can provide any object they wish to access in the callback method.|
+|`callback`|function||When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`, which is an [`AsyncResult`](/javascript/api/office/office.asyncresult) object.<br/><br/>On success, the item identifier is provided in the `asyncResult.value` property.|
+
+##### Errors
+
+|Error code|Description|
+|------------|-------------|
+|`ItemNotSaved`|The id can't be retrieved until the item is saved.|
```

Comment:
```suggestion
|`ItemNotSaved`|The ID can't be retrieved because the item has not yet been saved.|
```

Reply:
I believe this is the string that is actually returned as the error message.